Homework_7/Task_3.py

```diff
@@ -0,0 +1,17 @@
+numbers = input("Enter numbers by space: ").split()
+
+all_unique = True
+
+for i in range(len(numbers)):
+    for j in range(i + 1, len(numbers)):
+        if numbers[i] == numbers[j]:
+            all_unique = False
+            break
+    if not all_unique:
+        break
+
+if all_unique:
```

Comment:
Can be better)
![image](https://github.com/vithaiduchyk/hillel_python_basic/assets/51053483/b2a2cbf5-1cee-48eb-af8f-2040c636015b)

Reply:
I'll fix it